packages/logger/lib/logger.js

```diff
@@ -338,6 +338,14 @@ class Logger {
 				sanitizedLogData.message = null;
 			}
 
+			if (sanitizedLogData.error && sanitizedLogData.error instanceof Error) {
+				sanitizedLogData.error = serializeError(sanitizedLogData.error);
```

Comment:
**issue:** we'll need to update the documentation in the README for this too. [In this section](https://github.com/Financial-Times/dotcom-reliability-kit/tree/main/packages/logger#log-data-serialization) we say:

> Errors found in sub-properties of the log data are _not_ serialized like this. This is for performance reasons: looping over every nested property to check if it's an error is expensive.

We'll need to update to say that we _do_ automatically serialize a top-level `error` property but no others.

Reply:
Good point- thanks for spotting! I need to make a mental note to remember to check if the docs need to be updated when I submit a PR 😅